src/commands/BootstrapCommand.js

```diff
@@ -204,8 +204,8 @@ export default class BootstrapCommand extends Command {
         .forEach((dependency) => {
           // get Package of dependency
           const dependencyPackage = this.packageGraph.get(dependency).package;
-          // get path to dependency and its package.json
-          const dependencyLocation = dependencyPackage.location;
+          // get path to dependency and its scope
+          const { location: dependencyLocation, scope: dependencyScope } = dependencyPackage;
```

Comment:
This seems like it's getting more complicated than it needs to be...

`dependencyPackage.location` is where the module actually _is_.

`path.join(dependentPackage.nodeModulesLocation, dependencyPackage.name)` is where it needs to get _linked to_.

Seems like we have everything we need without running a regexp over anything?